packages/@internationalized/date/src/DateFormatter.ts

```diff
@@ -13,7 +13,7 @@
 let formatterCache = new Map<string, Intl.DateTimeFormat>();
 
 interface ResolvedDateTimeFormatOptions extends Intl.ResolvedDateTimeFormatOptions {
-  hourCycle?: Intl.DateTimeFormatOptions['hourCycle']
+  hourCycle?: NonNullable<Intl.DateTimeFormatOptions['hourCycle']>
```

Comment:
This looks like it was because TS was missing this property at one point, but looks like it exists now. We can probably just use Intl.ResolvedDateTimeFormatOptions directly instead

Reply:
Makes sense, done 😉